mongodb/src/main/kotlin/com/example/mongodb/config/SpringMongoConfig.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.mongodb.config
+
+import com.mongodb.client.MongoClients
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.data.mongodb.MongoDatabaseFactory
+import org.springframework.data.mongodb.core.MongoTemplate
+import org.springframework.data.mongodb.core.SimpleMongoClientDatabaseFactory
+
+@Configuration
+class SpringMongoConfig (
```

Comment:
Can do all this in your `application.yaml` if you want

Reply:
Spring should create the beans below for you by default I believe

Reply:
How do you mean? I thought I needed to use the `@Value` to bring in the properties from `application.yaml` and then create the classes? Is it with a certain annotation?